### DISCUSSION TITLE
Turn on/off path drawing

### DISCUSSION THREAD

**Discussion by TomasMiskov** [gpx-animator/gpx-animator#735] [Q&A]
Hey, is it possible to somehow pause the path drawing but keep the position of the point changing? In effect I would like to draw a part of my path, then pause the drawing but keep my movement visible and then restart the path drawing at which point a straight line would connect the last drawn point of the path and the new (re)-beginning - is this possible? Tips on how to achieve this? Cheers!

**Comment by playsbyear** [comment]
I think you can do something close to that by manipulating your GPX file. Cut out that middle part that you don't want to draw, and save it as a separate GPX file (and save the original file with the middle part cut out). Set the line width for the middle part file to zero, and then you'll just see the marker for it, but no drawing. And the original file without the middle section will cause that desired straight line between the drawing parts. I think the only undesirable thing will be that the marker for the middle part will remain visible at the end of the middle part.
